Scellecs.Morpeh/Unity/Utils/Editor/WorldBrowser/Windows/WorldBrowserWindow.cs

```diff
@@ -0,0 +1,97 @@
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+namespace Scellecs.Morpeh.Utils.Editor {
+    public class WorldBrowserWindow : EditorWindow {
+        private const string VIEW_CONTAINER_STYLE_SHEET = "Packages/com.scellecs.morpeh/Unity/Utils/Editor/WorldBrowser/Styles/ViewContainer.uss";
+        private const string HIERARCHY_SEARCH_STYLE_SHEET = "Packages/com.scellecs.morpeh/Unity/Utils/Editor/WorldBrowser/Styles/HierarchySearch.uss";
+        private const string HIERARCHY_STYLE_SHEET = "Packages/com.scellecs.morpeh/Unity/Utils/Editor/WorldBrowser/Styles/Hierarchy.uss";
+        private const string INSPECTOR_STYLE_SHEET = "Packages/com.scellecs.morpeh/Unity/Utils/Editor/WorldBrowser/Styles/Inspector.uss";
```

Comment:
These won't work unless Morpeh is in Packages folder? This is mostly never the case if it's installed via manifest file as it will be in Library/PackageCache/...